### DISCUSSION TITLE
Forward suricata events do external syslog server

### DISCUSSION THREAD

**Discussion by bit1290** [Security-Onion-Solutions/securityonion#6443] [Unsupported Versions]
Hello,

We need to forward Suricata events to a syslog server, but we can't accomplish that task.
We've tried the following:

1. Checked this URL https://docs.securityonion.net/en/2.3/logstash.html#forwarding-events-to-an-external-destination but we only have an example for zeek;
2. We tried to create our own:
[root@papidss003 custom]# cd /opt/so/saltstack/local/salt/logstash/pipelines/config/custom/
[root@papidss003 custom]# ls
suricata.conf
[root@papidss003 custom]# cat suricata.conf
filter {
  if [module] =~ "SuricataIDPS" and [dataset] =~ "common" {
    clone {
        id => "clone_zeek_dns_events"
        clones => ["zeek-dns-clone"]
        add_tag => [ "clone" ]
    }
  }
}
output {
  if "clone" in [tags] {
    tcp {
      id => "cloned_events_out"
      host => "10.135.101.131
      port => 514
      codec => "json_lines"
    }
  }
}
[root@papidss003 custom]#
[root@papidss003 custom]# cat /opt/so/saltstack/local/pillar/logstash/manager.sls
{%- set PIPELINE = salt['pillar.get']('global:pipeline', 'redis') %}
logstash:
  pipelines:
    manager:
      config:
        - so/0009_input_beats.conf
        - so/0010_input_hhbeats.conf
        - so/0011_input_endgame.conf
        - so/9999_output_redis.conf.jinja
        - custom/suricata.conf
[root@papidss003 custom]#

However we did a tcpdump and it didn't send nothing to our syslog server.

Could you help us with this situation?

Thank you.

**Comment by weslambert** [comment]
Try using a `module` value of `suricata` instead.

**Comment by bit1290** [comment]
Hello,

Still doesn't work.
I've followed this link to set this up: https://docs.securityonion.net/en/2.3/logstash.html?highlight=external#forwarding-events-to-an-external-destination
This is my manager file /opt/so/saltstack/default/pillar/logstash/manager.sls:

[manager.txt](https://github.com/Security-Onion-Solutions/securityonion/files/7667583/manager.txt)


And this is my logstash suricata conf file /opt/so/saltstack/local/salt/logstash/pipelines/config/custom/suricata.conf:

[suricata2.txt](https://github.com/Security-Onion-Solutions/securityonion/files/7667586/suricata2.txt)


This is my filebeat config file /opt/so/conf/filebeat/etc/filebeat.yml:

[filebeat.txt](https://github.com/Security-Onion-Solutions/securityonion/files/7667587/filebeat.txt)

And this is my suricata config file /opt/so/conf/suricata/suricata.yaml:

[suricata.txt](https://github.com/Security-Onion-Solutions/securityonion/files/7667604/suricata.txt)


I think everything is correct. Is there something that I'm missing here?

Thanks a lot!
